src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -2675,6 +2699,23 @@ def managed_instance_create(
     kwargs['sku'] = _find_managed_instance_sku_from_capabilities(cmd.cli_ctx, kwargs['location'], sku)
     kwargs['subnet_id'] = virtual_network_subnet_id
 
+    if kwargs['location'] in ['southeastasia', 'brazilsouth', 'eastasia']:
```

Comment:
Does `kwargs['location']` need to consider case sensitivity? If necessary, could we convert it to lower case and then compare it~

Reply:
Yes, will do. Thanks